be/src/formats/parquet/group_reader.h

```diff
@@ -165,9 +171,13 @@ class GroupReader {
     // columns(index) use as dict filter column
     std::vector<int> _dict_column_indices;
     std::unordered_map<int, std::vector<std::vector<std::string>>> _dict_column_sub_field_paths;
+    std::unordered_map<SlotId, std::vector<ExprContext*>> _left_conjunct_ctxs_by_slot_id;
```

Comment:
Extract all conjuncts into a class leads to high cohesion

Reply:
It means _left_no_dict_filter_conjunct_by_slot_it, dict filter will push down and rewrite, others left, will rename _left_no_dict_filter_conjuncts_by_slot